draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -10599,6 +10601,35 @@ Content-Type: text/plain
 </t>
 </section>
 
+<section title="Application Handling of Field Names" anchor="underscore.in.fields">
+<t>
+   When an implementation makes HTTP fields available to an application, it
+   is often necessary to map field names into an application-specific name
+   space. Most commonly, this is done by changing all instances of "-" to
```

Comment:
I tripped over that as well. Maybe we should really mention CGI, or systems that map fields to environment variables.

Reply:
I'm a bit wary of mentioning CGI specifically, because this convention is copied by other systems (eg WSGI), and readers might think that they aren't vulnerable because they're not actually using CGI. 

Mentioning environment vars makes sense. I can also see saying that it's mostly server-side, but again, I don't want to artificially limit it.